keps/77-dynamically-sized-jobs/README.md

```diff
@@ -33,142 +33,250 @@
 
 ## Summary
 
-Enable dynamic sizing of Kueue jobs. For the MVP, we will only focus on supporting RayClusters with autoscaling enabled, but other resources that can benefit from dynamic sizing should be supported eventually.
+Enable dynamic resizing of Kueue-managed batch/v1.Job workloads by supporting in-place horizontal scaling of job parallelism. 
+The primary goal is to allow jobs to scale their pod count without requiring suspension and requeueing, improving scheduling efficiency and reducing disruption during resource fluctuations.
 
-See: [Support dynamically sized (elastic) jobs #77](https://github.com/kubernetes-sigs/kueue/issues/77)
+While the initial focus is on batchv1/Job type, the long-term goal is to generalize support for dynamically sized (elastic) workloads across other Kueue-integrated resource types like RayClusters.
```

Comment:
If possible, I suggest we focus on both batchv1/Job and RayCluster, because it is very common to enable autoscaling for RayCluster which was the initial motivation for this proposal. I'm happy to help review, test and implement anything specific to RayCluster if needed to make this happen.

Reply:
Updated, PTAL. 

Please review the remaining KEP, specifically "Phases for MVP," and suggest RayClusters MVP targets if needed.